include/ofi_list.h

```diff
@@ -97,6 +97,11 @@ static inline void dlist_remove_init(struct dlist_entry *item)
 	dlist_init(item);
 }
 
+#define dlist_front(head, type, container, member)		\
+	do {								\
+		container = container_of((head)->next, type, member);	\
+	} while(0)
```

Comment:
A missing space??? *gasp*

Reply:
You do realize I'm not the author of this patch, right?

Reply:
I realized that right after I sent it and was very disappointed that I didn't catch you in a mistake.